vanilla-framework/vanilla-framework-react
# Add card component with modifier

## Done

- Add card component with modifier

## QA

- Pull code
- Run `npm run start`
- View http://localhost:3000
- Verify that `.p-card` and `p-card--highlighted` display
